package.json

```diff
@@ -32,20 +32,22 @@
   },
   "main": "./lib/nodejs/lib/thrift",
   "engines": {
-    "node": ">= 0.2.4"
+    "node": ">= 0.12.0"
   },
   "dependencies": {
-    "node-int64": "~0.3.0",
-    "q": "1.0.x",
-    "ws": "~0.4.32"
+    "node-int64": "^0.4.0",
+    "q": "^1.0.0",
+    "ws": "^1.0.0"
```

Comment:
It should be  >=1.1.1. In this version most of security issues were fixed.

Reply:
@ledara1 I'm pretty sure this work was abandoned.  We've since moved most of the build targets to more recent distributions, but this file wasn't updated.  There is a new ubuntu-xenial docker image which, if you would be able to update the package.json file to match, would be useful.